README.md

```diff
@@ -62,7 +62,15 @@ If you want to use MinIO as storage (including the test data), do the following:
 4. Add a bucket `shared-lifecyle`
 5. Copy the folders in `data/shared-lifecycle` in this repository to the bucket
 6. In `application.yml`, uncomment the `minio` section.
-7. Now Armadillo will automatically connect to MinIO at startup. 
+7. Now Armadillo will automatically connect to MinIO at startup.
```

Comment:
might want to describe the 'local' version too?

Reply:
See the beginning of this chapter 😉